broadcast-monitoring/infrastructure/elemental/custom_resources/custom-resource-py/libs/medialive.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python
```

Comment:
Do we need ASL for this if repository is MIT-0?

Reply:
Good question. Will remove.

Reply:
ASL already exists in a number of files... leaving this as is.